components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -124,50 +155,64 @@ const InfoListItemRender: React.ForwardRefRenderFunction<unknown, InfoListItemPr
         statusColor,
         ListItemButtonProps = {} as MuiListItemButtonProps,
         /* eslint-enable @typescript-eslint/no-unused-vars */
-        ...otherListItemProps
+        ...otherProps
     } = props;
 
     const combine = useCallback(
-        (className: keyof InfoListItemClasses): string => clsx(defaultClasses[className], classes[className]),
+        (className: keyof InfoListItemClasses): string => cx(defaultClasses[className], classes[className]),
         [defaultClasses, classes]
     );
 
     const getIcon = useCallback((): JSX.Element | undefined => {
         if (icon) {
             return (
                 <ListItemAvatar style={{ minWidth: 'unset' }}>
-                    <Avatar className={combine(avatar ? 'avatar' : 'icon')}>{icon}</Avatar>
+                    <Icon
+                        statusColor={statusColor}
+                        iconColor={iconColor}
+                        avatar={avatar}
+                        iconAlign={iconAlign}
+                        className={combine('avatar')}
+                    >
+                        {icon}
+                    </Icon>
                 </ListItemAvatar>
             );
         } else if (!hidePadding) {
             return (
                 // a dummy component to maintain the padding
                 <ListItemAvatar style={{ minWidth: 'unset' }}>
-                    <Avatar className={clsx(defaultClasses.avatar, defaultClasses.invisible)} />
+                    <InvisibleIcon
```

Comment:
Is this really an invisible icon? I would come up with a better way to name and organize this. This should probably be a styled avatar with the opacity style added to get the "invisible" effect.